contributors/design-proposals/node/troubleshoot-running-pods.md

```diff
@@ -82,22 +81,16 @@ kubectl debug target-pod
 
 This creates an interactive shell in a pod which can examine and signal other
 processes in the pod. It has access to the same network and IPC as processes in
-the pod. It can access the filesystem of other processes by `/proc/$PID/root`.
-As is already the case with regular containers, Debug Containers can enter
-arbitrary namespaces of another container via `nsenter` when run with
-`CAP_SYS_ADMIN`.
+the pod. When [process namespace sharing](https://features.k8s.io/495) is
```

Comment:
the top line should make it more clear that  this creates a new container and runs an interactive shell inside that container . `This creates a new container and starts an interactive shell inside that container in a pod called target-pod which can examine and signal other processes in the pod`